In [6]:

import pandas as pd

# Create a string contianing the path to your data
PATH_CSV = "https://archive.ics.uci.edu/ml/machine-learning-databases/00484/tripadvisor_review.csv"

# Save the data to a variable named df_csv
df_csv = pd.read_csv(PATH_CSV)

df_csv.head()

User ID  Category 1  Category 2  ...  Category 8  Category 9  Category 10
0  User 1        0.93         1.8  ...        2.79        1.82         2.42
1  User 2        1.02         2.2  ...        2.63        1.86         2.32
2  User 3        1.22         0.8  ...        2.80        1.31         2.50
3  User 4        0.45         1.8  ...        2.96        1.57         2.86
4  User 5        0.51         1.2  ...        2.78        1.18         2.54

[5 rows x 11 columns]

In [7]:
# Create a string contianing the path to your data
PATH_CSV = "https://archive.ics.uci.edu/ml/machine-learning-databases/00262/turkiye-student-evaluation_generic.csv"

# Save the data to a variable named df_csv
df_csv = pd.read_csv(PATH_CSV)

df_csv.head()

instr  class  nb.repeat  attendance  difficulty  ...  Q24  Q25  Q26  Q27  Q28
0      1      2          1           0           4  ...    3    3    3    3    3
1      1      2          1           1           3  ...    3    3    3    3    3
2      1      2          1           2           4  ...    5    5    5    5    5
3      1      2          1           1           3  ...    3    3    3    3    3
4      1      2          1           0           1  ...    1    1    1    1    1

[5 rows x 33 columns]

In [ ]:
# Create a string contianing the path to your data
PATH_CSV = "https://archive.ics.uci.edu/ml/machine-learning-databases/00262/turkiye-student-evaluation_generic.csv"

# Save the data to a variable named df_csv
df_csv = pd.read_csv(PATH_CSV)

df_csv.head()

In [14]:

# Create a string contianing the path to your data
PATH_exl = "https://archive.ics.uci.edu/ml/machine-learning-databases/forest-fires/forestfires.csv"

# Save the data to a variable named df_csv
df_csv = pd.read_csv(PATH_exl)

df_csv.head()

X  Y month  day  FFMC   DMC     DC  ISI  temp  RH  wind  rain  area
0  7  5   mar  fri  86.2  26.2   94.3  5.1   8.2  51   6.7   0.0   0.0
1  7  4   oct  tue  90.6  35.4  669.1  6.7  18.0  33   0.9   0.0   0.0
2  7  4   oct  sat  90.6  43.7  686.9  6.7  14.6  33   1.3   0.0   0.0
3  8  6   mar  fri  91.7  33.3   77.5  9.0   8.3  97   4.0   0.2   0.0
4  8  6   mar  sun  89.3  51.3  102.2  9.6  11.4  99   1.8   0.0   0.0

In [19]:
# show what each column means.
df_csv.describe()


X           Y        FFMC  ...        wind        rain         area
count  517.000000  517.000000  517.000000  ...  517.000000  517.000000   517.000000
mean     4.669246    4.299807   90.644681  ...    4.017602    0.021663    12.847292
std      2.313778    1.229900    5.520111  ...    1.791653    0.295959    63.655818
min      1.000000    2.000000   18.700000  ...    0.400000    0.000000     0.000000
25%      3.000000    4.000000   90.200000  ...    2.700000    0.000000     0.000000
50%      4.000000    4.000000   91.600000  ...    4.000000    0.000000     0.520000
75%      7.000000    5.000000   92.900000  ...    4.900000    0.000000     6.570000
max      9.000000    9.000000   96.200000  ...    9.400000    6.400000  1090.840000

[8 rows x 11 columns]

**Attribute Information:**

1. X - x-axis spatial coordinate within the Montesinho park map: 1 to 9
2. Y - y-axis spatial coordinate within the Montesinho park map: 2 to 9
3. month - month of the year: 'jan' to 'dec'
4. day - day of the week: 'mon' to 'sun'
5. FFMC - FFMC index from the FWI system: 18.7 to 96.20
6. DMC - DMC index from the FWI system: 1.1 to 291.3
7. DC - DC index from the FWI system: 7.9 to 860.6
8. ISI - ISI index from the FWI system: 0.0 to 56.10
9. temp - temperature in Celsius degrees: 2.2 to 33.30
10. RH - relative humidity in %: 15.0 to 100
11. wind - wind speed in km/h: 0.40 to 9.40
12. rain - outside rain in mm/m2 : 0.0 to 6.4
13. area - the burned area of the forest (in ha): 0.00 to 1090.84
(this output variable is very skewed towards 0.0, thus it may make
sense to model with the logarithm transform).